### ISSUE TITLE
ft_printf won't compile

### ISSUE THREAD

**Issue by skipper-83** [xicodomingues/francinette#18]
Hello there and thanks for making an amazing framework!
I have an issue with your printf-tester - tests 1-3 compile and work, but your own tests fail to compile. At first I thought there was a problem with my own makefile, but I can't seem to see what it is - and the errors I do see pop up in code that is not my own. The compiler throws 70+ errors mostly in this category: `[bonus.c:90:74: warning: format specifies type 'char *' but the argument has type 'void *' [-Wformat]] `- discarding the Werror flag in makes these warnings of course, but I still get "Problem compiling the tests" in the end. (execution.log attached)
Again - this might very well be my own fault in some way, but as the errors point in another direction I thought I might bring this to your attention.

Kind regards,
Albert van Andel

[execution.log](https://github.com/xicodomingues/francinette/files/9905403/execution.log)

**Comment by skipper-83** [comment]
Update on this: I delved a bit into your code and compiled my code and your tests with the lines from your makefile - compiles without issue (and manage to pass al strict tests after a long evening ;). - problem still persists in the main framework though

**Comment by xicodomingues** [comment]
Hey! This seem to be a problem in the newer versions of the compiler. I'll try to fix it in the future.

**Comment by skipper-83** [comment]
Wonderful! Happy to provide source code that invoke the problem if or when you get to it :)

**Comment by skipper-83** [comment]
Oh they included printf(“%%%”); in the latest update of Moulinette, would be a nice one to add :)

**Comment by xicodomingues** [comment]
`printf(“%%%”);` is an invalid format, and to me this should not be tested. I find it weird that they added this test case to Moulinette

**Comment by skipper-83** [comment]
I agree! I think nothing that compiles with warnings with printf should be tested, but it is hard to argue with La Moulinette

**Comment by skipper-83** [comment]
Hey there!

Problem persists, but I might have a pointer to a solution as I have the same problem with minitalk. It goes away when I comment out cleaning libft.a in my makefile, so there is some issue with that. Perhaps it is ar or the messages it generates? 

Kind regards,
Albert van Andel
